<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>passBox</class>
 <widget class="QDialog" name="passBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iw utility requires password</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="passLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passEntry">
     <property name="toolTip">
      <string extracomment="linssid uses the iw utility to scan for local wifi networks. The iw utility must run with sudo to do this, so your password is needed to continue."/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="passBtnBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>passBtnBox</sender>
   <signal>accepted()</signal>
   <receiver>passBox</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passBtnBox</sender>
   <signal>rejected()</signal>
   <receiver>passBox</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
